discordbot/economy/feargreed.py

```diff
@@ -68,12 +64,12 @@ async def feargreed_command(ctx, indicator=""):
             embed.set_image(url=uploaded_image.link)
 
         else:
-            if cfg.DEBUG:
-                print("Error with uploading the the image to Imgur.")
+            logger.error("Error when uploading the the image to Imgur.")
 
         await ctx.send(embed=embed)
 
     except Exception as e:
+        logger.error("ERROR Economy: [CNN] Feargreed. %s", e)
```

Comment:
Why is error only logged in this file and not in the other files of the economy directory?